src/app/route/ProductPage/ProductPage.component.js

```diff
@@ -196,13 +205,22 @@ class ProductPage extends Component {
                           areDetailsLoaded={ areDetailsLoaded }
                           configurableVariantIndex={ configurableVariantIndex }
                         />
-                        <ProductActions
-                          product={ dataSource }
-                          availableFilters={ filters }
-                          configurableVariantIndex={ configurableVariantIndex }
-                          areDetailsLoaded={ areDetailsLoaded }
-                          updateConfigurableVariantIndex={ index => this.updateUrl(index) }
-                        />
+                        <div>
```

Comment:
Please pay attention to accessibility. What does this div represent?

Reply:
This div is needed to display grouped products together with add to cart button. I added aria-label to identify what this div is for.